v0.10.6: "No rule to make target '../.version'" for static build

I just checked out the 0.10.6 release and used the instructions for the static build.
The build fails when executing `make -C contrib/static-builder-x86_64 build-crun CRUN_SOURCE=$(pwd)` with the following error:

```
[...]
configure: creating ./config.status                                                                                                                                    
config.status: creating Makefile                                                                                                                                        
config.status: creating config.h                                                                                                                                        
config.status: config.h is unchanged                                                                                                                                   
config.status: executing libtool commands                                                                       
config.status: executing depfiles commands                                                                      
 cd .. && /bin/sh /crun/build-aux/missing automake-1.16 --foreign Makefile                                       
 cd . && /bin/sh ./config.status Makefile depfiles                                                              
config.status: creating Makefile                                                                                 
config.status: executing depfiles commands                                                                       
make: *** No rule to make target '../.version', needed by 'all'.  Stop.  
make: *** [Makefile:9: build-crun] Error 2
```

--- a/Makefile.am
+++ b/Makefile.am
@@ -111,8 +111,8 @@ TESTS = tests/test_capabilities.py \
 dist-hook:
 	$(AM_V_GEN)echo $(VERSION) > $(distdir)/.tarball-version
 
-EXTRA_DIST += $(TESTS) tests/tests_utils.py build-aux/git-version-gen $(top_srcdir)/.version
-BUILT_SOURCES = $(top_srcdir)/.version
+EXTRA_DIST += $(TESTS) tests/tests_utils.py build-aux/git-version-gen .version
+BUILT_SOURCES = .version
 
 if HAVE_MD2MAN
 man1_MANS = crun.1

